go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/qri-io/dataset v0.1.5-0.20200324184139-108a69072ede
 	github.com/qri-io/deepdiff v0.1.1-0.20200305020550-8173efebcaa1
 	github.com/qri-io/doggos v0.1.0
-	github.com/qri-io/ioes v0.1.0
+	github.com/qri-io/ioes v0.1.1-0.20200407033451-28794e5a4c83
```

Comment:
I see that we mostly cut a release on ioes updates since they are so infrequent. Considering the last addition actually introduced functionality maybe cut it again and pin to that. Regardless, fine with both options.

Reply:
totally. I'll cut 0.1.1 & add